package.json

```diff
@@ -17,7 +17,7 @@
     "@mui/x-data-grid": "^6.11.0",
     "eslint": "latest",
     "eslint-config-next": "latest",
-    "next": "13.x",
+    "next": "^13.5.6",
```

Comment:
is this needed for this change?

Reply:
I have confirmed it isn't necessary, can drop. Just wanted to add a little more specificity to the next version. Do we want to bump to next 14?